packages/wonder-blocks-icon-button/src/components/__tests__/icon-button-types.test.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from "react";
+import {render, screen} from "@testing-library/react";
+
+import magnifyingGlassIcon from "@phosphor-icons/core/regular/magnifying-glass.svg";
+
+import {IconButton} from "../icon-button";
+import {IconButtonRef} from "../../util/icon-button.types";
+
+/**
+ * Tests for props that are common to the different IconButton types.
+ */
+describe("IconButton types", () => {
+    describe.each([{name: "IconButton", Component: IconButton}])(
```

Comment:
Yayy the shared component tests are catching on! I like it for making sure similar components fulfill the same API and have the same functionality 😄